x-pack/plugins/infra/public/plugin.ts

```diff
@@ -257,7 +257,7 @@ export class Plugin implements InfraClientPluginClass {
           defaultMessage: 'Inventory',
         }),
         path: '/inventory',
-        navLinkStatus: AppNavLinkStatus.visible,
+        navLinkStatus: this.isServerlessEnv ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden,
```

Comment:
What do you think of exporting this to a variable and reusing it below?
```
const navLinkStatus = this.isServerlessEnv ? AppNavLinkStatus.visible : AppNavLinkStatus.hidden
```
```suggestion
        navLinkStatus,
```

Reply:
`~~navLinkStatus~~` maybe something like `serverlessOnlyNav` would be a better name?

Reply:
Good point, will do 🙌

Reply:
I went with `serverlessNavLinkStatus`, hope it makes sense as well